apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/AbstractClaimRdaSink.java

```diff
@@ -46,6 +56,20 @@
   /** Holds the underlying value of our sequence number gauges. */
   private static final NumericGauges GAUGES = new NumericGauges();
 
+  /** Used to write out RDA messages to json strings */
+  protected static final JsonFormat.Printer writer =
+      JsonFormat.printer().omittingInsignificantWhitespace();
+
+  /** Used to map basic objects to json strings */
+  protected static final ObjectMapper mapper =
+      JsonMapper.builder()
+          .enable(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY)
+          .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+          .addModule(new Jdk8Module())
+          .addModule(new JavaTimeModule())
+          .serializationInclusion(JsonInclude.Include.NON_NULL)
+          .build();
+
```

Comment:
The serialization options here are close to what is being used in PACA-557 but not quite the same. Is that by design? If not, I think extracting this and using a standardized object mapper can save some headaches later.

Reply:
The serialization into postgres could be the same in both cases, in which case we discussed and there could be a common utility used in the `bfd-model-rda` module for postgres related transforms.  Being that 557 is waiting for `bfd-db-migrator`, we could merge this for now and fix it in 557 later if that works.